docker/README.MD

```diff
@@ -1,22 +1,34 @@
-## How-to build and run Docker image locally
+## Docker-compose
+
+Notes:
+-  run default (`docker-compose.yml`) docker-compose configuration with `docker-compose up` inside _docker/_ directory
+-  run other configurations by specifying `-f <config file>` argument. E.g:
+ `docker-compose  -f docker-compose-hostdb.yml up`
+-  every docker-compose run command takes env variables from `.env` file and injects them into the config file.
+-  in each docker-compose configuration there are default values which are used in case of not providing env variables. E.g. `${DATASOURCE_USERNAME:-openboxes}`
+
+Docker-compose config structure:
+-  `docker-compose-generic.yml` - generic configuration which every other docker-compose config extends. It defines generic OpenBoxes `app` and `nginx` services.
+-  `docker-compose.yml` - default config, which expects all env variables to be set in `.env` file or via command line. E.g. it requires proper definition of `DATASOURCE_URL` variable.
+-  `docker-compose-containerdb.yml` - it extends generic configuration with `db` service - MySQL docker container.
+-  `docker-compose-hostdb.yml` - it extends generic configuration with database connection to host's MySQL instance (via `host.docker.internal`).
+
+## Docker - build and run image from local Dockerfile
 
 ### Prerequisities:
 -  Docker 20.04+ is installed.
 -  OpenBoxes database (and db user) is already created.
--  Grails 3 is installed (e.g. with SDK Man).
 
 ### Instructions:
 
 **Note**: Run the below commands in **main** project directory.
 
 
-1. Run grails build to produce executable openboxes.war file. This WAR file contains embedded Tomcat.
-`grails prod war`
-2. Prepare docker build with gradle. It copies war and configuration files to *build/docker*.
-`./gradlew prepareDocker`
-3. Build a docker image:
+1. Prepare docker build with gradle. It runs grails build to produce executable openboxes.war file. This WAR file contains embedded Tomcat. It also copies war and configuration files to *build/docker*.
```

Comment:
Save yourself the trouble and use the markdown syntax for numbered list.

Reply:
I had some issues with the markdown numbered list not following the order when the text was indented - I'll try to see if this can be amended.

Reply:
From my experience you need to intend the code within a bulleted list (sometimes 4 spaces works, sometimes it requires)

1. Here's the first point with some code
        
        here's teh code

1. Point two without code